packages/gatsby-legacy-polyfills/.gitignore

```diff
@@ -0,0 +1,3 @@
+/dist/*.js
```

Comment:
I think it's missing an `.npmignore` that contains `__tests__` and `__fixtures__` (and perhaps a few others).

Reply:
i'm using `files` key in package.json